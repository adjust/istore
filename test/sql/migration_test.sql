BEGIN;
-- isagg should be upgradeable;
-- ./spec/istore/migration_spec.rb:4;
CREATE EXTENSION istore VERSION '0.1.3';
ALTER EXTENSION istore UPDATE TO '0.1.4';
ALTER EXTENSION istore UPDATE TO '0.1.3';
SELECT e.extname, e.extversion FROM pg_catalog.pg_extension e WHERE e.extname = 'istore'ORDER BY 1;;
ROLLBACK;
BEGIN;
-- isagg should be downgradeable;
-- ./spec/istore/migration_spec.rb:12;
CREATE EXTENSION istore;;
ALTER EXTENSION istore UPDATE TO '0.1.3';;
SELECT e.extname, e.extversion FROM pg_catalog.pg_extension e WHERE e.extname = 'istore'ORDER BY 1;;
ROLLBACK;
