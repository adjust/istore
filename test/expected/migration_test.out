BEGIN;
CREATE EXTENSION istore VERSION '0.1.3';
ALTER EXTENSION istore UPDATE TO '0.1.4';
ALTER EXTENSION istore UPDATE TO '0.1.3';
DROP EXTENSION istore;
CREATE EXTENSION istore;
ALTER EXTENSION istore UPDATE TO '0.1.3';
ROLLBACK;
